swap/src/bin/swap.rs

```diff
@@ -358,11 +360,21 @@ where
         let minimum_amount = bid_quote.min_quantity;
         let maximum_amount = bid_quote.max_quantity;
 
+        let code = QrCode::new(deposit_address.to_string())?;
+        let qr_code = "\n".to_string()
+            + code
+                .render::<unicode::Dense1x2>()
+                .dark_color(unicode::Dense1x2::Light)
+                .light_color(unicode::Dense1x2::Dark)
+                .build()
+                .as_str();
+
         info!(
             %deposit_address,
             %current_maximum_giveable,
             %minimum_amount,
             %maximum_amount,
+            %qr_code,
```

Comment:
Having the QRCode part of the tracing statement is a bit weird as that will completely break the JSON layout of the logs I believe.

Can we just print this to stderr instead and omit it entirely if we are in json format (ie --json has been passed as an argument)?

That should also address @rishflab 's comment on the horizontal alignment.

Reply:
Yeah I never thought about the --json use case that would make sense. I moved the QR Code printing to stderr.